src/views/Home/components/Schedule/schedule.scss

```diff
@@ -12,26 +12,28 @@
 
   &__schedule {
     position: absolute;
-    margin: auto;
-    left: 25%;
-    bottom: 25%;
-    width: 50%;
-    min-width: 300px;
+    left: 50%;
+    margin-left: -32%;
+    bottom: 40%;
+    width: 75%;
+    min-width: 500px;
```

Comment:
I'd apply a relative unit to this min-width using `calc` so components remain elastic. 
![Screenshot from 2021-01-13 18-15-51](https://user-images.githubusercontent.com/19658882/104535623-62d85980-55cb-11eb-9474-6d6c873b5a28.png)